DEVELOPER_GUIDE.md

```diff
@@ -20,3 +24,12 @@ To run the integration tests locally, run:
 The first argument, `opensearch` tells the server type to run integration test against. Valid values are `opensearch` and `opendistro`. The second argument tells whether to run server with security plugin enabled or not.
 
 Note that integration tests require docker to be installed and running, and downloads quite a bit of data from over the internet and hence take few minutes to complete.
+
+## Build the Documentation with Sphinx
+This are the steps to build the documentation with [Sphinx](https://www.sphinx-doc.org/):
+
+1. change into the `opensearch-py` directory where `setup.py` is located
+2. install opensearch-py - we recommend [editable mode](https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-e): `pip install -e .[docs]`
+3. change into the `docs` directory
+4. execute `make html`
```

Comment:
When running `make html` for the first time I noticed it gave a lot of warnings regarding not finding files. Every run after the first those warnings are gone. Is this because sphinx is trying to find license.txt but they are being manually hyperlinked?

Reply:
I fixed this by replacing it with real HTML links. This also fixes many broken links on this page: https://pypi.org/project/opensearch-py/

Reply:
Honestly I am in favor of doing fully qualified URL's. Honestly, that can wait until after the PR for this I think.